### ISSUE TITLE
Can you explain the references of the dual license?

### ISSUE THREAD

**Issue by seesemichaelj** [infusion/Complex.js#32]
Both in the `README.md` and `package.json`, there are mentions of a dual license of MIT or GPL2+, however `LICENSE` only contains MIT. There's also no mention to what the dual licensing is (i.e. under what conditions is it which license).

So I'm personally interpreting the license as "whichever the consumer would prefer", which feels redundant since MIT is much more permissive than GPL2.

Can you clarify this more? I appreciate it!

**Comment by infusion** [comment]
Thanks for bringing this up. Personally, I am totally in favor of MIT as everybody should be free to use FOSS in whatever way they like, as long as the original copyright remains intact. 

Your interpretation is right with "whichever the consumer would prefer" and also that GPL ⊆ MIT. The problem I saw with single license MIT is/(was?) that when getting embedded into a GPL project. Back then I saw this when Drupal (GPL) tried to use jQuery (MIT), so jQuery decided to dual license the project and so I followed this decision with my projects to give users all freedom. In the meantime, jQuery dropped the GPL license again, but I did not look into the reasoning yet.
